Fix include path for jbig libraries
Use CC, CFLAGS, OPT from EB.
Drop bad make depend lines.

John Dey, 2019-05-29  jfdey@fredhutch.org  fizwit@github.com
diff -ruN jbigkit-2.1.orig/Makefile jbigkit-2.1/Makefile
--- jbigkit-2.1.orig/Makefile	2014-03-27 11:47:15.000000000 -0700
+++ jbigkit-2.1/Makefile	2019-05-29 15:44:37.367937000 -0700
@@ -1,13 +1,9 @@
 # Unix makefile for JBIG-KIT
 
-# Select an ANSI/ISO C compiler here, GNU gcc is recommended
-CC = gcc
-
 # Options for the compiler: A high optimization level is suggested
-CFLAGS = -O2 -W -Wno-unused-result
-# CFLAGS = -O -g -W -Wall -Wno-unused-result -ansi -pedantic # -DDEBUG
+INCLUDE = -I../libjbig
+CFLAGS = $(OPT) $(INCLUDE)
 
-export CC CFLAGS
 
 VERSION=2.1
 
diff -ruN jbigkit-2.1.orig/pbmtools/Makefile jbigkit-2.1/pbmtools/Makefile
--- jbigkit-2.1.orig/pbmtools/Makefile	2014-03-27 11:47:15.000000000 -0700
+++ jbigkit-2.1/pbmtools/Makefile	2019-05-29 15:48:31.063713000 -0700
@@ -1,11 +1,9 @@
 # Unix makefile for the JBIG-KIT PBM tools
 
-# Select an ANSI/ISO C compiler here, e.g. GNU gcc is recommended
-CC = gcc
-
 # Options for the compiler
-CFLAGS = -g -O -W -Wall -Wno-unused-result -ansi -pedantic # --coverage
-CPPFLAGS = -I../libjbig 
+INCLUDE = -I../libjbig
+CFLAGS = $(OPT) $(INCLUDE)
+CPPFLAGS = $(OPT) $(INCLUDE)
 
 .SUFFIXES: .1 .5 .txt $(SUFFIXES)
 .PHONY: txt test test82 test85 clean
